.github/workflows/release-tests.yml

```diff
@@ -31,3 +31,31 @@ jobs:
           . venv2\scripts\activate
           echo "import distutils.util  # pylint: disable=unused-import" > test.py
           pylint test.py
+
+  additional-dependencies-linux-tests:
+    name: Regression tests w/ additional dependencies (Linux)
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v3.0.0
+      - name: Set up Python
+        id: python
+        uses: actions/setup-python@v3.1.0
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+      - name: Install Qt
+        run: |
+          sudo apt-get install build-essential libgl1-mesa-dev
```

Comment:
As mentioned over in https://github.com/PyCQA/astroid/pull/1531#discussion_r860683187:

> I'm confused - why does "Install Qt" actually only do `sudo apt-get install build-essential libgl1-mesa-dev`? You are installing PyQt from binary wheels, so that 40s step can probably just be dropped entirely.